examples/custom-interactions/webpack.config.js

```diff
@@ -2,7 +2,7 @@
 const {resolve} = require('path');
```

Comment:
I think the destructuring has been avoided in other examples to support older versions of node, maybe copy it?